configuration/config-txt/audio.md

```diff
@@ -6,6 +6,10 @@ The onboard audio output uses config options to change the way the analogue audi
 
 By default, a 1.0LSB dither is applied to the audio stream if it is routed to the analogue audio output. This can create audible background "hiss" in some situations, for example when the ALSA volume is set to a low level. Set `disable_audio_dither` to `1` to disable dither application.
 
+### enable_audio_dither
```

Comment:
It was there, in the previous paragraph. I've reordered to make it a bit clearer.